web/html/static_files.go

```diff
@@ -134,3 +139,96 @@ func (ar *Router) ResetPasswordHandler(pathComponents ...string) http.HandlerFun
 		}
 	}
 }
+
+// LoginHandler serves login page or redirects to the callback_url if user is already authenticated.
+func (ar *Router) LoginHandler(pathComponents ...string) http.HandlerFunc {
```

Comment:
file called static_files, this handler is not looks  like static file, I believe it deserves it's own login dedicated file